src/components/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+export const HomePage: React.FC = () => {
+  return (
+    <h1 className="title">Home page</h1>
+  );
```

Comment:
Would be better to wrap it in div container, because the page is not a title text